src/handwritten/_install_daqmx.py

```diff
@@ -0,0 +1,332 @@
+import click
+import contextlib
+import errno
+import json
+import logging
+import os
+import pathlib
+import subprocess
+import sys
+import tempfile
+import traceback
+from typing import List, Tuple, Optional
+import urllib.request
+
+if sys.platform.startswith("win"):
+   import winreg
+
+MODULE_DIR = pathlib.Path(__file__).parent
+METADATA_FILE = MODULE_DIR / "metadata.json"
```

Comment:
This makes assumptions about how this package is deployed.

https://docs.python.org/3/library/importlib.resources.html has APIs for accessing files embedded in packages.

Reply:
ok